projects/plugins/jetpack/modules/comments/comments.php

```diff
@@ -711,17 +711,10 @@ public function add_comment_meta( $comment_id ) {
 	 * @return boolean
 	 */
 	public function should_show_subscription_modal() {
-
-		// Not allow it to run on self-hosted or simple sites
```

Comment:
Regardless of the comment, this was disabled only at Jetpack sites.

The code reads "IF NOT simple or atomic site, OR NOT simple site, don't enable modal".